main.go

```diff
@@ -74,6 +72,9 @@ func main() {
 
 	sseType := s3manager.SSEType{Type: viper.GetString("SSE_TYPE"), Key: viper.GetString("SSE_KEY")}
 
+	viper.SetDefault("TIMEOUT", 600)
+	serverTimeout := time.Duration(viper.GetInt32("TIMEOUT")) * time.Second
```

Comment:
Please move the reading of the `TIMEOUT` environment variable up to where all the other environment variables are read.